service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/post-startup.sh

```diff
@@ -88,6 +88,8 @@ if [[ -n "$TERRA_SSH_KEY" ]]; then
   sudo -u "$JUPYTER_USER" sh -c 'ssh-add .ssh/id_rsa; ssh-keyscan -H github.com >> ~/.ssh/known_hosts'
 fi
 
+sudo -u "${JUPYTER_USER}" sh -c "pip install pandas_gbq"
```

Comment:
nit: can we do it next to other common package installation.

Reply:
Ah, I missed the `conda install`. Moved, thanks.